app/controllers/general.php

```diff
@@ -566,32 +566,6 @@
         );
     });
 
-App::get('/manifest.json')
```

Comment:
why was this removed?

Reply:
This should be handled by the console, not appwrite 👍🏻 

If at some point we are going to add it for a PWA - we will do it there.